docs/start/smeshing/smeshing_basic/advanced_config.md

```diff
@@ -3,15 +3,19 @@ id: advanced_config
 title: Advanced Smapp Configuration
 ---
 
-
-While running Smapp, you should see two config files in the Smapp data directory (by default, `$HOME/.config/Spacemesh` on Linux):
+After running Smapp, you should see two config files in the Spacemesh data directory:
```

Comment:
Not true, it's a smapp directory not Spacemesh. The original was also correct this one not. As the original mentioned actually a smapp configs while "spacemesh data directory" most likely means node data dir which is not the one here :)